docs/index.css

```diff
@@ -46,3 +46,55 @@ code {
 .Demo ul {
   margin: var(--space-large);
 }
+
+.Footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  flex: none;
+  flex-direction: row;
+  box-sizing: border-box;
+  z-index: var(--z-index-sidebar);
+  display: flex;
+  align-items: center;
+  height: 50px;
```

Comment:
Fixed heights can cause issues with zooming/variable text sizes. Instead, it would be better to only set the appropriate padding so the element scales with the text OR set the size relative to the font size.

Reply:
😬 I agree, but this came from the UXPin.

Reply:
You can still achieve the same height relative to the text by using the appropriate top/bottom padding to still end up with 50px.

Reply:
🤔 The TopBar height is fixed too. I have been repeatedly told to follow the UXPin - Would it be worth working with Design to get a responsive design and work on that in another PR/issue?

Reply:
When the browser is used at a smaller screen size like `360` width, the height of the `.Footer-links` is taller than the `height: 50px` `.Footer` container. I think you could remove the explicit height and it would be fine?

Reply:
To get it to be `50px` height at a larger browser width, it would be `padding: 0.359375rem 0;.

Reply:
🤔 sidebar breaks into it at that width when using `0.36rem` padding

![Screen Shot 2022-09-15 at 3 09 59 PM](https://user-images.githubusercontent.com/99676396/190489202-f5225bc4-7cba-42be-8dc4-484f1f06a66a.png)

Reply:
The `bottom` property would need to adjust for the `.Sidebar-with-footer` since the footer needs to be taller at that screen width.

Reply:
I was hoping I would get more math from you 😆

Reply:
Added!

Reply:
I don't see the 1px gray border on top of footer when in the sidebar area.

<img width="158" alt="Screen Shot 2022-09-15 at 3 29 05 PM" src="https://user-images.githubusercontent.com/3081483/190512105-39f6b22a-4e83-499f-99c0-bd71a403e5e9.png">

Can we switch the bottom rem to `bottom: 3.5325rem;`?

<img width="209" alt="Screen Shot 2022-09-15 at 3 31 02 PM" src="https://user-images.githubusercontent.com/3081483/190512382-5f94f0f1-ea4e-485c-983e-d714f9f8de07.png">